packages/state-manager/src/sagas/communities/updateCommunityMetadata/updateCommunityMetadata.saga.ts

```diff
@@ -37,6 +39,17 @@ export function* sendCommunityMetadataSaga(
   yield* apply(
     socket,
     socket.emit,
-    applyEmitParams(SocketActionTypes.SEND_COMMUNITY_METADATA, communityMetadataPayload)
+    applyEmitParams(
+      SocketActionTypes.SEND_COMMUNITY_METADATA,
+      communityMetadataPayload,
+      (response: CommunityMetadata) => {
+        responseChan.put(response)
+      }
+    )
```

Comment:
This demonstrates the socket.io acknowledgement feature on the frontend. It's a little funny here since redux saga uses generator functions and so apparently one way to make this work is to use a channel to pass the result from the response handler to the main function.